frontend/packages/integration-tests-cypress/tests/crud/annotations.spec.ts

```diff
@@ -115,7 +115,7 @@ describe('Annotations', () => {
     modal.shouldBeClosed();
 
     cy.log('Verify first two saved annotations');
-    cy.byTestID('edit-annotations').contains('2 annotation'); // 3rd annotation without key should not have been saved
+    cy.byTestID('edit-annotations').contains('2 annotations'); // 3rd annotation without key should not have been saved
```

Comment:
This isn't a bad change, but I'm not sure it fixes the flake? If `2 annotations` matches, `2 annotation` would also match since it's a substring check

Reply:
From what I've read it seems like `cy.contains` should work that way.  But clearly it didn't.   Changed assertion method, it may help.
![image](https://user-images.githubusercontent.com/12733153/106796292-28416a00-6629-11eb-8651-7f063d492e4b.png)

Reply:
@dtaylor113 Could it be a race where we check the annotation text before the watch updates the page (then the page updates after the failure but before the screenshot)?

If it were the substring problem, I'd expect the test to fail every time.

Reply:
I'm not sure what cypress can wait upon if that is the case, other than an arbitary `wait(5000)`.   I thought `.contains()` continually checks the DOM as it is returning a DOM element.   I was leaning more towards perhaps `.contains()` doesn't do substring, rather full word 'contains', or perhaps the space in the middle, idk.

Reply:
`.should('have.text', '2 annotations')` is more precise.

Reply:
> I was leaning more towards perhaps `.contains()` doesn't do substring, rather full word 'contains', or perhaps the space in the middle, idk.

@dtaylor113 If that were the cause, wouldn't it fail every time? It only fails about 1 time in 20, which suggests a race to me.

Reply:
Might have something to do with https://docs.cypress.io/guides/core-concepts/retry-ability.html#Only-the-last-command-is-retried